Mathlib/MeasureTheory/Measure/MutuallySingular.lean

```diff
@@ -53,6 +53,23 @@ theorem mk {s t : Set α} (hs : μ s = 0) (ht : ν t = 0) (hst : univ ⊆ s ∪
   exact subset_toMeasurable _ _ hxs
 #align measure_theory.measure.mutually_singular.mk MeasureTheory.Measure.MutuallySingular.mk
 
+/-- A set such that `μ h.nullSet = 0` and `ν h.nullSetᶜ = 0`. -/
+def nullSet (h : μ ⟂ₘ ν) : Set α := h.choose
+
+lemma measurableSet_nullSet (h : μ ⟂ₘ ν) : MeasurableSet h.nullSet := h.choose_spec.1
+
+@[simp]
+lemma measure_nullSet (h : μ ⟂ₘ ν) : μ h.nullSet = 0 := h.choose_spec.2.1
+
+@[simp]
+lemma measure_compl_nullSet (h : μ ⟂ₘ ν) : ν h.nullSetᶜ = 0 := h.choose_spec.2.2
+
+@[simp]
+lemma restrict_nullSet (h : μ ⟂ₘ ν) : μ.restrict h.nullSet = 0 := by simp
+
+@[simp]
+lemma restrict_compl_nullSet (h : μ ⟂ₘ ν) : ν.restrict h.nullSetᶜ = 0 := by simp
```

Comment:
I don't think those should be simp lemmas, as they can be proved directly by simp.

Reply:
It indeed looks like they should not be simp lemmas for that reason, but simp does not use them if I don't tag them, and the linter does not complain.
I'll add a note about that strange situation.